docs/ruleset_functions/get_zone_BPF_BAT.md

```diff
@@ -0,0 +1,142 @@
+# get_zone_BPF_BAT
+**Schema Version:** 0.0.29
+
+**Description:** Get a dictionary of the BPF_BAT and areas for a given zone.
+- The function looks at the space lighting type to guess a building area type for a BuildingSegment whose bpf_building_area_type has not been specified AND lighting_building_area_type is NONE
+
+**Inputs:**  
+- **B-RMD**: The baseline ruleset model instance
+- **zone_id**: the id of the zone
+
+**Returns:**  
+- **zone_BPF_BAT_dict**: A dict for the zone that saves the BPF_BAT as keys and the areas as the values. Example: {SCHOOL: 50000, UNDETERMINED: 2000}
+ 
+**Function Call:** 
+- None
+
+## Logic:  
+
+- create a dictionary to map lighting space types to BPF building area types: ```building_area_lookup = { 
+	ATRIUM_LOW_MEDIUM: UNDETERMINED,							
+	ATRIUM_HIGH: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_AUDITORIUM: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_CONVENTION_CENTER: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_EXERCISE_CENTER: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_GYMNASIUM: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_MOTION_PICTURE_THEATER: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_PENITENTIARY: ALL_OTHER,
+	AUDIENCE_SEATING_AREA_PERFORMING_ARTS_THEATER: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_RELIGIOUS_FACILITY: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_SPORTS_ARENA: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_TRANSPORTATION_FACILITY: UNDETERMINED,
+	AUDIENCE_SEATING_AREA_ALL_OTHER: UNDETERMINED,
+	BANKING_ACTIVITY_AREA: UNDETERMINED,
+	CLASSROOM_LECTURE_HALL_TRAINING_ROOM_PENITENTIARY: ALL_OTHER,
+	CLASSROOM_LECTURE_HALL_TRAINING_ROOM_SCHOOL: SCHOOL,
+	CLASSROOM_LECTURE_HALL_TRAINING_ROOM_ALL_OTHER:	UNDETERMINED,
+	CONFERENCE_MEETING_MULTIPURPOSE_ROOM: UNDETERMINED,
+	CONFINEMENT_CELLS: ALL_OTHER,
+	COPY_PRINT_ROOM: UNDETERMINED,
+	CORRIDOR_FACILITY_FOR_THE_VISUALLY_IMPAIRED: UNDETERMINED,
+	CORRIDOR_HOSPITAL: HEALTHCARE_HOSPITAL,
+	CORRIDOR_MANUFACTURING_FACILITY: ALL_OTHER,
+	CORRIDOR_ALL_OTHERS: UNDETERMINED,
+	COURT_ROOM:	ALL_OTHER,
```

Comment:
I think courts are typically classfied as offices so maybe we can consider OFFICE here